carma_cloud_client/launch/carma_cloud_client_launch.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2022 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+from ament_index_python import get_package_share_directory
+from launch import LaunchDescription
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.descriptions import ComposableNode
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
+from carma_ros2_utils.launch.get_current_namespace import GetCurrentNamespace
+
+import os
+
+
+'''
+This file is can be used to launch the CARMA carma_cloud_client_node.
+  Though in carma-platform it may be launched directly from the base launch file.
+'''
+
+def generate_launch_description():
+
+    # Declare the log_level launch argument
+    log_level = LaunchConfiguration('log_level')
+    declare_log_level_arg = DeclareLaunchArgument(
+        name ='log_level', default_value='WARN')
```

Comment:
Is it possible to have this log_level set in the parameters.yml file to allow easy update  or this here is the same as yaml file in terms of updating it at runtime?

Reply:
I dont think launch file can read and change its values based on yaml file. But we can change the values in launch file easily without needing to rebuild